cli/tests/integration_tests.rs

```diff
@@ -2085,6 +2085,35 @@ mod integration {
       }
     }
 
+    #[cfg(unix)]
+    #[test]
+    fn pty_complete_declarations() {
+      use std::io::Read;
+      use util::pty::fork::*;
+      let deno_exe = util::deno_exe_path();
+      let fork = Fork::from_ptmx().unwrap();
+      if let Ok(mut master) = fork.is_parent() {
+        master.write_all(b"class MyClass {}\n").unwrap();
+        master.write_all(b"My\t\n").unwrap();
+        master.write_all(b"let myVar;\n").unwrap();
+        master.write_all(b"myV\t\n").unwrap();
+        master.write_all(b"close();\n").unwrap();
+
+        let mut output = String::new();
+        master.read_to_string(&mut output).unwrap();
+
+        assert!(output.contains("> MyClass"));
+        assert!(output.contains("> myVar"));
+
+        fork.wait().unwrap();
+      } else {
+        std::env::set_var("NO_COLOR", "1");
+        let err = exec::Command::new(deno_exe).arg("repl").exec();
+        println!("err {}", err);
+        unreachable!()
+      }
```

Comment:
It seems this conditional is repeated in every repl test... 

I'm in favor of moving REPL tests to separate file as you suggested @dsherret , `integrations_tests.rs` became unwieldy at this point.

Reply:
Yeah, I just copied what was done before. It would be nice to extract all this out then clean it up... after all the repl changes/fixes I will confirm it should be done then do it.